spec/graphql/object_type_spec.rb

```diff
@@ -17,6 +17,20 @@
     }
   end
 
+  it "doesn't allow invalid name" do
+    exception = assert_raises(GraphQL::InvalidNameError) {
+      InvalidNameObject = GraphQL::ObjectType.define do
+        name "Three Word Query"
+
+        field :id, !types.Int, "id field"
+      end
+
+      # Force evaluation
+      InvalidNameObject.name
+    }
+    assert_equal("Names must match (?-mix:^[_a-zA-Z][_a-zA-Z0-9]*$) but 'Three Word Query' does not", exception.message)
```

Comment:
If you call `#inspect` on the regexp and interpolate that instead then you won’t get the `(?-mix:…)` part.

Reply:
Good point, I thought maybe its useful to have it there, but i agree it's cleaner with `inspect`! thanks.